packages/aws-rfdk/lib/lambdas/lambdaLayerVersionArns.ts

```diff
@@ -3,5 +3,108 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/* eslint-disable */
-export const ARNS = {"openssl-al2":{"ap-northeast-2":"arn:aws:lambda:ap-northeast-2:224375009292:layer:openssl-al2:2","ca-central-1":"arn:aws:lambda:ca-central-1:224375009292:layer:openssl-al2:2","eu-north-1":"arn:aws:lambda:eu-north-1:224375009292:layer:openssl-al2:2","eu-west-1":"arn:aws:lambda:eu-west-1:224375009292:layer:openssl-al2:2","eu-west-3":"arn:aws:lambda:eu-west-3:224375009292:layer:openssl-al2:2","us-west-2":"arn:aws:lambda:us-west-2:224375009292:layer:openssl-al2:2","ap-northeast-1":"arn:aws:lambda:ap-northeast-1:224375009292:layer:openssl-al2:2","ap-south-1":"arn:aws:lambda:ap-south-1:224375009292:layer:openssl-al2:2","ap-southeast-1":"arn:aws:lambda:ap-southeast-1:224375009292:layer:openssl-al2:2","ap-southeast-2":"arn:aws:lambda:ap-southeast-2:224375009292:layer:openssl-al2:2","eu-west-2":"arn:aws:lambda:eu-west-2:224375009292:layer:openssl-al2:2","us-east-1":"arn:aws:lambda:us-east-1:224375009292:layer:openssl-al2:2","us-east-2":"arn:aws:lambda:us-east-2:224375009292:layer:openssl-al2:2","ap-northeast-3":"arn:aws:lambda:ap-northeast-3:224375009292:layer:openssl-al2:1","eu-central-1":"arn:aws:lambda:eu-central-1:224375009292:layer:openssl-al2:2","sa-east-1":"arn:aws:lambda:sa-east-1:224375009292:layer:openssl-al2:2","us-west-1":"arn:aws:lambda:us-west-1:224375009292:layer:openssl-al2:2"}};
+import {
+  ILayerVersion,
+  LayerVersion,
+} from '@aws-cdk/aws-lambda';
+import {
+  CfnMapping,
+  Construct,
+  Lazy,
+  Stack,
+} from '@aws-cdk/core';
+
+export enum LambdaLayer {
+  OPEN_SSL_AL2 = 'OpenSslAl2',
+}
+
+type RegionMap<T> = { [region: string]: T };
+type LambdaLayerVersionArnMap<T> = { [layer: string]: T };
+
+const arns: LambdaLayerVersionArnMap<RegionMap<string>>= {
+  [LambdaLayer.OPEN_SSL_AL2]: {
```

Comment:
These ARNs are autogenerated during the release publish of the RFDK -- https://github.com/aws/aws-rfdk/blob/mainline/lambda-layers/bin/write-ts.ts

We can't hard-code them in the construct like this.

Reply:
Ah, I didn't notice that. I'll get another revision up to address that